app/controllers/resources_controller.rb

```diff
@@ -38,8 +38,44 @@ def suggestions
     render json: ToolFilterService.new(params).call, include: params[:include], fields: field_params, status: :ok
   end
 
+  def publish_translation
+    if valid_publish_params?
+      publish_translations
+      head :no_content
```

Comment:
@cesarediaz from the sample request & response I had provided this method should return the `Translation` objects that were created.